### PR TITLE
troubleshoot: refine doc TOC

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
troubleshoot: refine doc org

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.4 (TiDB 6.4 versions)
- [x] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: TOC.md [hunk 1]
```diff
@@ -172,23 +172,27 @@
...
-  - [写冲突与写性能下降](/troubleshoot-write-conflicts.md)
-  - [磁盘 I/O 过高](/troubleshoot-high-disk-io.md)
-  - [锁冲突与 TTL 超时](/troubleshoot-lock-conflicts.md)
-  - [TiFlash 常见问题](/tiflash/troubleshoot-tiflash.md)
-  - [数据索引不一致报错](/troubleshoot-data-inconsistency-errors.md)
+  - 故障诊断问题汇总
+    - [TiDB 集群问题导图](/tidb-troubleshooting-map.md)
+    - [TiDB 集群常见问题](/troubleshoot-tidb-cluster.md)
+    - [TiFlash 常见问题](/tiflash/troubleshoot-tiflash.md)
+  - 部分故障场景
```
Comment (TomShawn) [inline TOC.md:179]:
```suggestion
  - 故障场景
```

File: tidb-troubleshooting-map.md [hunk 2]
```diff
@@ -165,6 +165,10 @@ aliases: ['/docs-cn/dev/tidb-troubleshooting-map/','/docs-cn/dev/how-to/troubles
 
     解决方法：可以通过手动加 `Cast(xx as decimal(a, b))` 来绕过这个问题，a 和 b 就是目标的精度。
 
+### 3.5 慢查询问题
+
+要定位慢查询，参阅[慢查询日志](/identify-slow-queries.md)
```
Comment (TomShawn) [inline tidb-troubleshooting-map.md:170]:
```suggestion
要定位慢查询，参阅[慢查询日志](/identify-slow-queries.md)。要处理慢查询，参阅[分析慢查询](/analyze-slow-queries.md)。
```

File: TOC.md [hunk 3]
```diff
@@ -172,23 +172,27 @@
...
+      - [定位慢查询](/identify-slow-queries.md)
+      - [分析慢查询](/analyze-slow-queries.md)
+    - [TiDB OOM 故障排查](/troubleshoot-tidb-oom.md)
+    - [热点问题处理](/troubleshoot-hot-spot-issues.md)
+    - [CPU 占用过多导致读写延迟增加](/troubleshoot-cpu-issues.md)
+    - [写冲突与写性能下降](/troubleshoot-write-conflicts.md)
+    - [磁盘 I/O 过高](/troubleshoot-high-disk-io.md)
+    - [锁冲突与 TTL 超时](/troubleshoot-lock-conflicts.md)
+    - [数据索引不一致报错](/troubleshoot-data-inconsistency-errors.md)
+  - 故障诊断工具
```
Comment (lilin90) [inline TOC.md:190]: 对“工具”的说法有点疑虑，改成“方法”怎么样？
```suggestion
  - 故障诊断方法
```

**Review by lilin90 — APPROVED** [review]
LGTM

**Pull request merged**